### DISCUSSION TITLE
NOT received  education benefits within 5 days of approval status

### DISCUSSION THREAD

**Discussion by Nin-zhihao** [community/community#141520] [GitHub Education] [answered]
### Hi everyone,

Hi, everyone!
My application has been approved for 5 days, but I'm still unable to access the benefits. Can someone assist me?
<img width="244" alt="屏幕截图 2024-10-15 153637" src="https://github.com/user-attachments/assets/c6868957-816f-451e-93a4-c4e1ab8fd62f">

**Comment by davevad93** [comment, marked as answer]
Hi @Nin-zhihao ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Nin-zhihao was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Nin-zhihao** [reply]
OK, I'll wait for the best.
